CMakeLists.txt

```diff
@@ -312,7 +313,11 @@ foreach (family ${ARCH})
 
     # Link in awooter, if enabled.
     if (AWOOTER)
-        target_link_libraries(${PROGRAM_PREFIX}nextpnr-${family} PRIVATE awooter)
+        include_directories(common/route/awooter 
+            common/kernel/ common/place/ common/route/ "${family}/")
```

Comment:
I'm not entirely sure about this; doesn't this duplicate the `aux_source_directories`? https://github.com/YosysHQ/nextpnr/blob/fc01f13cfde4e128c3a4c651636ade328d8d5eae/CMakeLists.txt#L257-L259

Reply:
Most likely! I truly have no idea what I am doing with CMake.